src/CoreWCF.WebHttp/src/Resources/Strings.resx

```diff
@@ -375,6 +375,9 @@
   <data name="SerializingRequestNotSupportedByFormatter" xml:space="preserve">
     <value>The formatter '{0}' does not support serializing requests.</value>
   </data>
+  <data name="ServiceModelWebServiceNotRegistered" xml:space="preserve">
+    <value>Service Model Web Service not registered, call IServiceCollection.AddServiceModelWebServices() in Startup.cs.</value>
```

Comment:
This is a little obscure, and not quite accurate. The error message should say what the problem is (WebHttpBehavior endpoint behavior is missing) and suggest a possible solution. Also a Startup class isn't necessary, or it could be called something different. This would be a better error message:
```
Unable to find the endpoint behavior WebHttpBehavior for endpoint address "{0}". This is possibly caused by not calling  ServiceModelWebServiceCollectionExtensions.AddServiceModelWebServices(IServiceCollection services).
```
Then in the code, passing address as a parameter to SR.Format.